### DISCUSSION TITLE
would have been nice to include a list of the results for those who don't have the time to set this up.

### DISCUSSION THREAD

**Discussion by softyoda** [PlummersSoftwareLLC/Primes#889] [Q&A]
As stated in the title

**Comment by rbergen** [comment]
Maybe you should have a look at https://plummerssoftwarellc.github.io/PrimeView/? That includes the results from a number of systems (including 3 from Dave), that are updated on a daily basis.

**Reply by softyoda** [reply]
Hi, thanks for the link, I can't found the way/column to compare languages, the page only displays computers benchmarks.

**Comment by rbergen** [comment]
Click on the date/time field of any row on the first page to go to the result page for that run.
